assets/css/tw.css

```diff
@@ -380,9 +321,16 @@
 .no-js\:block {
   display: none;
 }
+.no-js\:flex {
+  display: flex;
+}
```

Comment:
To make this work, you also need to add the inverse style to partial/header/noscript.html:

```
  .no-js\:flex {
    display: flex;
  }
```